"""
Represents an exception when user authentication fails
"""
class CredentialsException(Exception):
    pass
